test/units/module_utils/common/text/formatters/test_human_to_bytes.py

```diff
@@ -182,3 +182,75 @@ def test_human_to_bytes_isbits_wrong_default_unit(test_input, unit, isbits):
     """Test of human_to_bytes function, default_unit is in an invalid format for isbits value."""
     with pytest.raises(ValueError, match="Value is not a valid string"):
         human_to_bytes(test_input, default_unit=unit, isbits=isbits)
+
+
+@pytest.mark.parametrize(
+    'test_input,expected',
+    [
+        ('10 BBQ sticks please', "can't interpret following string"),
+        ('3000 GB guns of justice', "can't interpret following string"),
+        ('1 EBOOK please', "can't interpret following string"),
+        ('3 eBulletins please', "can't interpret following string"),
+        ('1 bBig family', "can't interpret following string"),
+    ]
+)
+def test_human_to_bytes_nonsensical_inputs_first_two_letter_unit(test_input, expected):
+    """Test of human_to_bytes function to ensure it raises ValueError for nonsensical inputs that has the first two
+    letters as a unit."""
+    with pytest.raises(ValueError, match=expected):
+        human_to_bytes(test_input)
```

Comment:
```suggestion
@pytest.mark.parametrize(
    'test_input',
    [
        ('10 BBQ sticks please'),
        ('3000 GB guns of justice'),
        ('1 EBOOK please'),
        ('3 eBulletins please'),
        ('1 bBig family'),
    ]
)
def test_human_to_bytes_nonsensical_inputs_first_two_letter_unit(test_input):
    """Test of human_to_bytes function to ensure it raises ValueError for nonsensical inputs that have the first two
    letters as a unit."""
    expected = "can't interpret following string"
    with pytest.raises(ValueError, match=expected):
        human_to_bytes(test_input)
```

Same for the below tests as well.